app/views/articles/index.html.erb

```diff
@@ -6,4 +6,4 @@
     <% end %>
   </ul>
 </div>
-<a href="/articles/new">Make a new article</a>
\ No newline at end of file
+<%= link_to "Make a new article", new_article_path, class: "new_article" %>
```

Comment:
Interesting! I would phrase this a little bit differently, but the basic idea is right. I believe it's not the syntax that you're used to seeing for CSS class selectors because it's not being applied in HTML, but provided as part of an argument to the `link_to` method.

[The docs](http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to) list the third parameter for `link_to` as `html_options`, which generally will allow you to pass a hash with `class` and/or `id`.

I haven't tried the following, but recommend you do:

```erb
<%= link_to("Make a new article", new_article_path, {class: "new_article"}) %>
```

That, to me at least, makes the method call a little easier to parse. Let me know if that *doesn't* work and we can think through this some more.